systemcontractindex/stakingindex/bucket.go

```diff
@@ -106,17 +109,20 @@ func assembleVoteBucket(token uint64, bkt *Bucket, contractAddr string, blockInt
 	if bkt.UnlockedAt != maxBlockNumber {
 		vb.StakeStartBlockHeight = bkt.UnlockedAt
 	}
+	if bkt.Muted {
+		vb.Candidate, _ = address.FromString(address.ZeroAddress)
```

Comment:
as discussed, check where `getAllBuckets()` are used, make sure the muted bucket won't be counted

Reply:
also, i think should not change the candidate? since a muted bucket is still a valid bucket, it just won't be counted for votes
the caller should check the muted flag
```
if bkt.Muted { do not count it for votes}
```

Reply:
I prefer to keep the mute functionality hidden within the indexer, as I haven't found any need to expose it at the moment. If there is a need in the future, we can adjust it later; this only involves changes to memory.